### DISCUSSION TITLE
[Resolved] Starting November 22, 2023, replies to questions and other inquiries will be delayed

### DISCUSSION THREAD

**Discussion by tatsuya6502** [moka-rs/moka#351] [Announcements]
I am the author of Moka and the sole maintainer at this time. I will be delayed in replying to inquiries for the next few weeks because I will be in a hospital 🏥 for medical treatments. I will have some chances to check GitHub Issues and Discussions from there, but will not actively respond to them.

Sorry for the inconvenience. I am hoping my personal life will settle down soon.

* * *

**EDIT** Nov 29, 2023

I am back from the hospital, and I feel so much better now!
https://github.com/moka-rs/moka/discussions/351#discussioncomment-7703532

**Comment by abusch** [comment]
Mate, as much as we all appreciate your work on this great crate, this is Open Source: you work on it for free, so you don't owe us anything! Your personal life 100% comes first, no need to apologise :)

Take care, and hope you get better soon!

**Comment by tatsuya6502** [comment]
Hi. Thanks everybody for you supports. I am back from the hospital. I am expecting to resume development of Moka soon.

I was discharged from the hospital this morning. I had surgery on a coronary artery in my heart. The surgery improved the blood flow to my heart, making my heart healthier and improving blood flow throughout my body. The general discomfort I was feeling for the past few years seems to be getting better already.

For the next few weeks, I will have to prioritize my full time job. But, now I am confident that I will be able to resume development of Moka as well.
